src/components/engage/Engage.vue

```diff
@@ -59,7 +59,21 @@
         <label for="share-my-score">
           Yes, I want to share my score on the Engagement Index
         </label>
-        <v-btn v-if="!$currentUser.isAdmin && !$currentUser.isTeaching && $currentUser.sharePoints === null" type="submit">Continue</v-btn>
+        <v-btn
+          v-if="!$currentUser.isAdmin && !$currentUser.isTeaching && $currentUser.sharePoints === null"
+          id="continue-btn"
+          :disabled="isSaving"
+          @click="saveSharePoints"
```

Comment:
@johncrossman If switching to @click, do we also need @keydown.enter for accessibility?

Reply:
I think we do. Amending.